ovh/provider_test.go

```diff
@@ -396,6 +396,11 @@ func testAccPreCheckOkmsCredential(t *testing.T) {
 	checkEnvOrSkip(t, "OVH_OKMS_CREDENTIAL")
 }
 
+func testAccPreCheckVrack(t *testing.T) {
```

Comment:
can be removed, there is already the `testAccPreCheckVRack` func that does the same thing